Documentation/benchmarks/etcd-2-2-benchmarks.md

```diff
@@ -0,0 +1,62 @@
+## Physical machines
```

Comment:
As I mentioned before, please compare the result with 2.1 and tell people why there is a change very briefly.